test/scraped/response/decorator/absolute_urls_test.rb

```diff
@@ -13,6 +13,9 @@
     <a class="external-link" href="http://example.org/person-123-other-page">Person 123</a>
     <a class="empty-link" href="">Person 123</a>
     <a class="javascript-link" href="javascript:chooseStyle('small',60);">Person 123</a>
+    <a class="bracketed-link" href="/person[123]">Person 123</a>
+    <a class="relative-link-with-unencoded-space" href="/person 123">Person 123</a>
+    <a class="encoded-url" href="/person%20fred">Person Fred</a>
```

Comment:
Again, this is inconsistent with all the other tests, which will lead anyone reading it to assume that there's something substantially different going on here. If there's a reason why this can't also be person 123, then you should make that clear, otherwise you should maintain the consistency.